4회 4주차/부족한 금액 계산하기.js

```diff
@@ -0,0 +1,10 @@
+function solution(price, money, count) {
+  let answer;
+  if (money < price * ((count * (count + 1)) / 2)) {
+    answer = price * ((count * (count + 1)) / 2) - money;
+  } else {
+    answer = 0;
+  }
+
+  return answer;
```

Comment:
그리구 return값을 음수가 되지 않도록 하게 작성하면 더 간단한 코드가 될것 같아요 !!! 
```suggestion
return Math.max(0, totalCost - money);
```